### PR TITLE
Careers page

### PR DESCRIPTION
**Pull request by Meghana-12**
Related to issue #396

View he page on route `/careersPage`
CareersPage

### CODE REVIEW

File: src/ignitus-CareersPage/styles.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import styled from '@emotion/styled';
+import * as C from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/colors';
+import * as F from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/fonts';
+import { StyleTypes} from './types';
+import {Heading4} from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/typography';
+import {minimumWidthQuery} from '../ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/media';
+
+export const Container = styled.div`
+  background-color: ${C.White};
```
Comment (Paarmita) [inline src/ignitus-CareersPage/styles.ts:9]: Remove this background color property, and leave to grey, that is present in other screens.

**Comment by Meghana-12** [comment]
> @Meghana-12 Thanks for the PR. Can you please make the page responsive to mobile and iPad, using the media queries defined, and, the alignment of the lower section with open positions is not in the center, please correct that as well.
> 
> Also, please solve the merge conflicts.

sure, I'll fix those.

File: src/ignitus-CareersPage/Components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+    </S.Wrapper>
+    <S.Wrapper>
+      {WhyContent.map((x: any) => (
+        <S.Paragraph>{x.content}<br/><br/></S.Paragraph>
+      ))};
+    </S.Wrapper>
+  </S.Section>
+);
+
+const BuildingSection = ({title,imgLink}: Basictypes) => (
```
Comment (Paarmita) [inline src/ignitus-CareersPage/Components/index.tsx:182]: In the open jobs, add a pointer cursor on hovering on the job.

File: src/ignitus-Routes/ignitus-PublicRoutes/index.tsx [hunk 3]
```diff
@@ -45,6 +48,7 @@ export const PublicRoutes: React.FunctionComponent = () => {
             component={AuthenticationRoutes}
           />
           <Route path="/profile" component={StudentProfile} />
+          <Route path="/careersPage" component={CareersPage}/>
```
Comment (Paarmita) [inline src/ignitus-Routes/ignitus-PublicRoutes/index.tsx:51]: Add this route in the footer for the `Career` as it would redirect to this page.

**Comment by Meghana-12** [comment]
> @megha Please resolve merge conflicts, I will review once conflicts are resolved, as its hard to figure which files have you changed due to conflicts.

sure!

File: src/ignitus-Routes/ignitus-PublicRoutes/index.tsx (hunk 3 above)
Comment (Meghana-12) [reply src/ignitus-Routes/ignitus-PublicRoutes/index.tsx:51]: okay

File: src/ignitus-CareersPage/styles.ts (hunk 1 above)
Comment (Meghana-12) [reply src/ignitus-CareersPage/styles.ts:9]: okay

**Pull request closed without merging**